doc/tutorial.rst

```diff
@@ -61,24 +61,24 @@ where :math:`x_i \in \mathcal{R}^{p \times 1}, i = \{1, 2, \dots, n\}` are
 the observed independent variables (predictors),
 :math:`\beta_0 \in \mathcal{R}^{1 \times 1}`,
 :math:`\beta \in \mathcal{R}^{p \times 1}`
-are linear coefficients. :math:`\lambda_i` is also known as the conditional
-intensity function, conditioned on :math:`(\beta_0, \beta)` and
+are linear coefficients. The rate parameter :math:`\lambda_i` is also known as
+the conditional intensity function, conditioned on :math:`(\beta_0, \beta)` and
 :math:`q(z) = \exp(z)` is the nonlinearity.
 
 For numerical reasons, let's adopt a stabilizing non-linearity, known as the
-`softplus` or the smooth rectifier `Dugas et al., 2001
-<http://papers.nips.cc/paper/1920-incorporating-second-order-functional-knowledge-for-better-option-pricing.pdf>`_,
-and adopted by Jonathan Pillow's and Liam Paninski's groups for neural data
+`softplus` or the smooth rectifier (see `Dugas et al., 2001
```

Comment:
I think it might be nicer to move these links to the bottom of the page so that the links don't interrupt the flow of the text in the rst. Like we do in the whats_new.rst file. What do you think? It's of course a "good to have" thing if you have time

Reply:
not high priority; resolving